package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "synthetic-monitoring-app",
-  "version": "0.0.7",
```

Comment:
We had some versioning weirdness with the last release, this is just playing catch up to the current version